pkg/imgpkg/registry/keychain_test.go

```diff
@@ -169,6 +170,58 @@ func TestAuthProvidedViaEnvVars(t *testing.T) {
 		}), auth)
 	})
 
+	for i, validHostname := range []string{
+		"localhost:9999",
+		"http://localhost:9999",
+		"https://localhost:9999",
+		"localhost:9999/v1/",
+		"localhost:9999/v2/",
+		"*:9999/v2/",
+		"local*:9999/v2/",
```

Comment:
can we throw in a test for multi domain globs: *.*.foo.com?

Reply:
yep done!